src/index.html

```diff
@@ -7,8 +7,49 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a class="logo" href="#">
+          <img src="images/logo.png"
+        alt="Moyo">
+        </a>
+        <nav class="nav">
+          <ul class="nav-list">
+            <li class="nav-link">
+              <a class="link is-active" href="#Apple">Apple</a>
+            </li>
+            <li class="nav-link">
+              <a class="link" href="#Samsung">Samsung</a>
+            </li>
+            <li class="nav-link">
```

Comment:
Add empty lines between multiline sibling blocks of HTML
```suggestion
          <ul class="nav-list">
            <li class="nav-link">
              <a class="link is-active" href="#Apple">Apple</a>
            </li>
            
            <li class="nav-link">
              <a class="link" href="#Samsung">Samsung</a>
            </li>
            
            <li class="nav-link">
```